src_new/util.js

```diff
@@ -830,15 +830,28 @@ exports.vLogInfo = function(divID, infoObject){
 	}
 };
 
-exports.getExternalBidderStatus = function() {
-	return window.OWT.externalBidderStatus;
+exports.getExternalBidderStatus = function(divIds) {
```

Comment:
This method can be moved OWT/PWT name-space as it is specific to a functionality